VERSION

```diff
@@ -1 +1 @@
-1.8.6
+1.8.7
```

Comment:
This is incremented on release only.
Our latest release is 1.8.6, so this should stay as is.